notifier/core/flaskapp.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
+from .private.singleton import Singleton
+from ..api.flask.endpoints import endpoint
+from .config import Config
+
+
+@Singleton
+class FlaskApp(Flask):
+    def __init__(self):
+        self.__config = Config.instance()
+        super().__init__(self.__config.key("app_name"))
+        self.register_blueprint(endpoint)
+
+    def start(self):
+        self.run()
```

Comment:
We need to add `thread=True` (I don't remember if this is the right parameter name) option to get a better performance.